### ISSUE TITLE
WinGet - Failed to install as Administrator (Hash Mismatch) 

### ISSUE THREAD

**Issue by ooobii** [dbeaver/dbeaver#15132]
#### System information: 
- **Operating system**: Windows 11 (Build 22000.376)
- **DBeaver version**: 21.3.0.202111281534 (Not Latest)
- **Additional extensions**: N/A

#### Connection specification:
- **Database name and version**: N/A
- **Driver name**: N/A
- **Do you use tunnels or proxies (SSH, SOCKS, etc)?**: N/A

#### Describe the problem you're observing:
When attempting to install / update DBeaver using `winget` command line utility, an error is encountered. The downloaded installer hash does not match the hash provided in DBeaver's `winget` repository entry.

#### Steps to reproduce, if exist:
1. Launch PowerShell / Windows Terminal as Administrator.
2. Execute `winget upgrade --id "dbeaver.dbeaver"` (or, in my case, `winget upgrade --all`)
3. `winget` downloads the installer successfully.
4. Installation is skipped because the installer hash did not match, and `winget` will not bypass hash checks when running as Administrator.

#### Include any warning/errors/backtraces from the logs
Here is a screenshot from the output of `winget`:
![image](https://user-images.githubusercontent.com/60083197/148563449-df1f0fb4-c75c-4a4a-8bfa-d1d53756a91e.png)

**Comment by mattcargile** [comment]
I used [chocolatey](https://www.chocolatey.org) and it did work. I used `choco install dbeaver -y`.

**Comment by ghost** [comment]
According to the official winget documentation, you could use the parameter `--force` for both `winget install` and `winget upgrade` commands. Works only if you don't run it as Administrator though.

**When a hash mismatch is discovered will ignore the error and attempt to install the package.**

https://docs.microsoft.com/en-us/windows/package-manager/winget/upgrade#options

**Comment by ooobii** [comment]
I want to preface with a thank you for your responses. 😺 Love me some DBeaver!!!

I'm aware that hash checks can be skipped, but skipping them is **not** a good idea / work around in stricter security environments. I am using this as part of an automated background script that runs as Administrator (so that UAC does not trigger when triggering certain installation packages). 

I need to verify the origin of all software downloaded & installed on machines that use this. Suggestions like "just install a package on your computer that failed a hash verification check" aren't a solution 😃

**Comment by ooobii** [comment]
> I used [chocolatey](https://www.chocolatey.org) and it did work. I used `choco install dbeaver -y`.

I will use this for the time being - hash checks in `choco` match! Thanks, @mattcargile 👍🏼 

But, the problem with `winget` still remains and should be corrected.

**Comment by Matvey16** [comment]
@ooobii Thank you for the report, this happened because we implemented a hotfix after a release but the version number remained the same. This should be fixed now, because version 21.3.3 was released

**Comment by ooobii** [comment]
I've confirmed that @Matvey16 is correct - hashes were correct with the latest version. Thank you for the background 😄!

I'll go ahead and close this issue.

**Comment by JRHille** [comment]
Sorry to reopen this thread, but is there any way to bypass the hash check with the window open as an admin?

I created a script to install the programs on my computer and I was having a Hash problem with the EA App, which is now fixed, I even made the change request and it is fixed.
But I would like to have an option to skip this check in case some other application has this problem again in the future. Unfortunately, I can't simulate the error again as it has already been fixed in the app in question, but I remember that I tried to use the --force command and it still didn't want to install.

From what I read in the comment above, this command does not work with the cmd window as an administrator, is this correct?
